### PR TITLE
Add mm fonts

### PR DESCRIPTION
**Pull request by boyoonc**

### CODE REVIEW

File: src/sass/bootstrap/modified-type.scss [hunk 1]
```diff
@@ -16,26 +16,26 @@ h1, h2, h3, h4, h5, h6,
...
 
 h2{
   font-family: $font-stack-serif;
+  font-weight: 500;
   line-height: 0.9;
 }
 
 h3{
-  font-weight: bold;
+  font-weight: 600;
```
Comment (besseym) [inline src/sass/bootstrap/modified-type.scss]: Good. The use of a numerical variable is better than `bold`.

File: src/sass/_custom-fonts.scss [hunk 2]
```diff
@@ -0,0 +1,350 @@
...
+//  src: url('../fonts/mm-fonts/32C37C_D_0.eot');
+//  src: url('../fonts/mm-fonts/32C37C_D_0.eot?#iefix') format('embedded-opentype'),url('../fonts/mm-fonts/32C37C_D_0.woff') format('woff'),url('../fonts/mm-fonts/32C37C_D_0.ttf') format('truetype');
+//}
+
+//BOOK
+@font-face {
+  font-family: 'FreightText';
+  font-weight: 400;
+  font-style: normal;
+  src: url('../fonts/mm-fonts/32C37C_8_0.eot');
```
Comment (besseym) [inline src/sass/_custom-fonts.scss]: Is this directory relative to mmui or to the project?

Comment (besseym) [reply src/sass/_custom-fonts.scss]: If this is relative to the project, perhaps this file doesn't belong in mmui.

Comment (besseym) [reply src/sass/_custom-fonts.scss]: Yeah, if the font paths are relative to the project, I'm thinking the font imports should happen in the project sass.

**Comment by boyoonc** [comment]
ready for reivew!

**Pull request merged**